### PR TITLE
Development and copy-edit complete; chapter seems done

### PR DESCRIPTION
**Pull request by quaid**
- Did some content integration; much of this has been copy edited through a few times.
- Adjusted some language and especially the paragraphing to make three authors appear more as one.
- Other formatting fixes, pronoun adjustments.
- Conclusion in place
- I'm calling this ready for Final Review, meaning I think it's done and ready for any other final reviews folks want to do.

Signed-off-by: Karsten Wade <kwade@redhat.com>

### CODE REVIEW

**Comment by quaid** [comment]
I'm going to hold this as a pull request for a few hours in case @palante is able to see it and wants to make any comments or adjustments before I land the branch.

Guedis—after that it's fine to keep fixing (even after release tomorrow), just this is a good in-flight moment for you.

**Review by palante — APPROVED** [review]

**Comment by palante** [comment]
This is great, thanks @quaid! :D

**Pull request merged**
